alf/algorithms/generator.py

```diff
@@ -246,13 +346,17 @@ def __init__(self,
                 critic update, used for ``minmax``.
             critic_optimizer (torch.optim.Optimizer): Optimizer for training the
                 critic, used for ``minmax``.
+            inverse_mvp_optimizer (torch.optim.Optimizer): Optimizer for training
+                the inverse_mvp network, used when ``functional_gradient`` is 
+                ``True``.
```

Comment:
is ``True`` => is "rkhs" ? Or perhaps we should make functional_gradient a boolean value?

Reply:
Will make into a boolean. We have a derivation and implementation for the `minmax` option, so there was this other option. But the minmax variant did not pass tests. I may work some more on the minmax implementation and PR it in the future. For now `functional_gradient` can be a `bool`